daemons/ipa-kdb/ipa_kdb_principals.c

```diff
@@ -49,6 +49,19 @@
                                       "(objectclass=krbprincipal))" \
                                     "(krbprincipalname=%s)" \
                                     "%s)"
+
+#define PRINC_TGS_SEARCH_FILTER_WILD "(&(|(objectclass=krbprincipalaux)" \
+                                     "(objectclass=krbprincipal)" \
+                                     "(objectclass=ipakrbprincipal))" \
+                                     "(|(ipakrbprincipalalias=*)" \
+                                     "(krbprincipalname=*)))"
+
+#define PRINC_TGS_SEARCH_FILTER_WILD_EXTRA "(&(|(objectclass=krbprincipalaux)" \
+                                          "(objectclass=krbprincipal)" \
+                                          "(objectclass=ipakrbprincipal))" \
+                                          "(|(ipakrbprincipalalias=*)" \
+                                          "(krbprincipalname==*))" \
+                                         "%s)"
```

Comment:
The aligment of the strings is a bit off. First line is indented one more than the rest, last line is indented one space less.